addons/auth_passkey_portal/static/src/js/passkeys_portal_create.js

```diff
@@ -0,0 +1,65 @@
+import { Interaction } from "@web/public/interaction";
+import { InputConfirmationDialog } from "@portal/js/components/input_confirmation_dialog/input_confirmation_dialog";
+import { registry } from "@web/core/registry";
+import { renderToMarkup } from "@web/core/utils/render";
+import { handleCheckIdentity } from "@portal/interactions/portal_security";
+import { _t } from "@web/core/l10n/translation";
+import * as passkeyLib from "@auth_passkey/../lib/simplewebauthn";
+import { user } from "@web/core/user";
+
+export class PortalPasskeyCreate extends Interaction {
+    static selector = "#portal_passkey_add";
+    dynamicContent = {
+        _root: { "t-on-click": this.onClick },
+    };
+
+    async onClick() {
```

Comment:
For cleanliness, shouldn't you follow a structure like
https://github.com/odoo/odoo/blob/2712e75a9f85847359b26eca7bcc5850e1aab2d2/addons/portal/static/src/interactions/portal_security.js#L16-L20

Rather than a generic "onClick" ?